### DISCUSSION TITLE
My Favorite bug so far!

### DISCUSSION THREAD

**Discussion by bbmaster123** [ALTaleX531/TranslucentFlyouts#95] [Show and tell]
![bestbug](https://github.com/ALTaleX531/TranslucentFlyouts/assets/92948468/a7b13e12-ae04-4c39-8e75-0ef646daab02)

I haven't seen this happen anywhere else, but this is what I see in the xbox app on windows 11. Doubt its a big deal, I actually laughed when I saw that!

cheers :)

**Comment by ALTaleX531** [comment]
Technically, I don't know why this bug can happen😰, but at least it's less destructive🤔 and brings you a little joy I guess...

**Comment by IdkWhatToCallMe123** [comment]
what the acutal fuck LMAO 💀
